testing/src/main/java/org/jdbi/v3/testing/JdbiAssertions.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.testing;
+
+import org.assertj.core.api.Assertions;
+
+/**
+ * Extension entry-point for custom JDBI assertions.
+ */
+public class JdbiAssertions extends Assertions {}
```

Comment:
this makes no sense to me. Why would this code extend an assertions class?

Reply:
Extending `Assertions` gives access to all AssertJ `assertThat` methods without the need for static imports.

Reply:
Why would static imports be bad? Inheritance is not intended for "make some methods available without import". These are public methods and now I can write "TestFoo.assertThat" in some other test class. 

JDBI uses inheritance where it makes sense. Most of the time, aggregation and encapsulation work better.